ecommerce_integrations/shopify/order.py

```diff
@@ -185,37 +186,46 @@ def _get_total_discount(line_item) -> float:
 	return sum(flt(discount.get("amount")) for discount in discount_allocations)
 
 
-def get_order_taxes(shopify_order, setting):
-	taxes = []
+def get_order_taxes(shopify_order, setting, items):
+	tax_account_wise_data = {}
 	line_items = shopify_order.get("line_items")
 
 	for line_item in line_items:
 		item_code = get_item_code(line_item)
 		for tax in line_item.get("tax_lines"):
-			taxes.append(
-				{
+			account_head = get_tax_account_head(tax)
+			tax_account_wise_data.setdefault(
+				account_head, {
 					"charge_type": "Actual",
-					"account_head": get_tax_account_head(tax),
```

Comment:
this can be kept for consistency. You are already keeping this in lines 289 and 308